documentation/src/docs/controller-service.md

```diff
@@ -517,275 +494,253 @@ the lock on the resource and hence permission to process the task. The
 parameters for executing a task are serialized and stored under the
 resource.
 
-Currently we use task framework only for create stream tasks. All the
-other background processing is done using event processor framework.
+Currently we use Task framework only to create Stream tasks. All the
```

Comment:
`Currently, we use the Task Framework...`.

Reply:
Accepted suggestion